src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py

```diff
@@ -709,8 +707,9 @@ def get_chart_name_and_version(
                 chart_version = data["version"]
             else:
                 raise FileOperationError(
-                    f"A name or version is missing from Chart.yaml in the helm package '{helm_package.name}'. \
-                        Please fix this and run the command again."
+                    "A name or version is missing from Chart.yaml in the helm package"
+                    f" '{helm_package.name}'.                         Please fix this"
```

Comment:
ditto

Reply:
Sorry, and thanks for catching. I think that was the merge from main messing things up :-(